experiments/conductor/cmd/runner/utilities.go

```diff
@@ -208,8 +208,12 @@ func gitAdd(ctx context.Context, workDir string, files ...string) error {
 }
 
 func gitCommit(ctx context.Context, workDir string, msg string) error {
-	log.Printf("COMMAND: git commit -m %q", fmt.Sprintf("conductor: %q", msg))
-	gitcommit := exec.CommandContext(ctx, "git", "commit", "-m", fmt.Sprintf("conductor: %q", msg))
+	authorName := "k8s-config-connector-conductor[bot]"
+	authorEmail := "conductor@example.com" // TODO: create a real robot account?
```

Comment:
maybe kcc-conductor@google.com ? or google groups.

Reply:
Yeah, I think there's a process for creating a group / account for GitHub use, but I'd like to hear others' opinions before moving forward with it.

Reply:
Either works for me as long as it passes the CLA?

Reply:
@jingyih - Lets get this change in. Do you want to create the email for the bot ?

Reply:
PR updated